### PR TITLE
adding my static page

### PR DESCRIPTION
**Pull request by ravitadi**

### CODE REVIEW

File: protfolio.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+h1 {
+    color: #1A73E8;
+}
+
+img {
+    border: 1px solid black;
```
Comment (ravitadi) [inline protfolio.css:6]: hey this color should be green

File: protfolio.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!Doctype html>
+<html>
+    <head>
+        <title>This is Ravi's profolio website</title>
```
Comment (ravitadi) [inline protfolio.html:4]: this code needs more meta tags for seo and others purposes.

File: protfolio.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            alt="This is a picture of charlie chaplin" />
+        <a href="www.linkedin.com">Click here to go to my Linkedin.</a>
+        <p class="descriptionOne"> Hello world, this is my portfolio website. 
+            Feel free to visit my other links in my website 
+        </p>
+        <p class="descriptionTwo">
+            This static website is built using html and css.
+        </p>
+    </body>
+</html>
```
Comment (ravitadi) [inline protfolio.html:21]: always add a new line end of end of every html file

File: protfolio.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+    width: 100px;
```
Comment (amelano) [inline protfolio.css:7]: This width should be 500px.

File: protfolio.css [hunk 5]
```diff
@@ -0,0 +1,8 @@
+h1 {
+    color: #1A73E8;
```
Comment (Egbertzen) [inline protfolio.css:2]: please change that in green

File: protfolio.html [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+<html>
+    <head>
+        <title>This is Ravi's profolio website</title>
+        <meta author="ravi" />
+        
+        <link rel="stylesheet" href="index.css" type="text/css" />
+    </head>
+    <body>
+        <h1>My Portfolio Website</h1>
+        <img src="https://cps-static.rovicorp.com/3/JPG_400/MI0004/070/MI0004070047.jpg"
```
Comment (Egbertzen) [inline protfolio.html:11]: add another image

**Pull request closed without merging**